source/indexes.txt

```diff
@@ -177,8 +179,8 @@ The following example creates a text index on the specified string field:
    :language: ruby
    :dedent:
 
-.. TODO: To learn more about text indexes, see the :ref:`ruby-text-index`
-.. guide.
+To learn more about text indexes, see the :ref:`ruby-text-index`
+guide.
```

Comment:
Can't comment on the line directly, but similarly, is the geospatial guide worth linking to in the geospatial section?